spec/link-spec.js

```diff
@@ -76,8 +112,8 @@ you should not [click][her]
       })
     )
 
-    it('does not open non http/https links', async () => {
-      await atom.workspace.open('sample.txt')
+    it('does not open non http/https/atom links', async () => {
+      await atom.workspace.open('sample.md')
```

Comment:
I updated the tests to use .md files since `language-gfm` is loaded and `language-text` is not